ts/Core/Tooltip.ts

```diff
@@ -1379,6 +1380,12 @@ class Tooltip {
 
         // single point tooltip
         } else {
+            // Hovering over series that tooltip shouldn't be split, #13868.
+            if (!isArray(pointOrPoints) && pointOrPoints.series.noSplitTooltip) {
+                tooltip.options.split = false;
+                tooltip.split = false;
+            }
```

Comment:
I'm afraid this might not work, because there's no reset to `split=true`. I wanted to try this demo: https://jsfiddle.net/BlackLabel/5wdz4a8y/1/ - but it fails